victor/src/layout/boxes/generation.rs

```diff
@@ -399,18 +400,31 @@ impl IntermediateBlockLevelBox {
 }
 
 impl DeferredNestedBlockContainer {
-    fn finish(self, context: &Context, style: &Arc<ComputedValues>) -> BlockContainer {
+    fn finish(
+        self,
+        context: &Context,
+        style: Arc<ComputedValues>,
+    ) -> (BlockContainer, Arc<ComputedValues>) {
```

Comment:
It looks like this would be simpler if `IntermediateBlockContainerBuilder` held a `&Arc<ComputedStyle>` instead?